var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -385,10 +386,11 @@ def nvariant_bool(feature):
 
         rendering = variant_bool("+opengl2", "OpenGL2", "OpenGL")
         includes = variant_bool("+development_files")
+        use_x11 = nvariant_bool("+osmesa") if not sys.platform == "win32" else "OFF"
```

Comment:
We should conditionally define the `+osmesa` variant if it should always be off on Windows.

Reply:
OK that makes sense. It looks like `osmesa` can in fact be used on Windows.

My only request here then is to use `spec.satisfies(platform=windows)` (vs. `sys.platform`).